common/changes/@microsoft.azure/openapi-validator-rulesets/fps_patchPropertiesCorrespondToPutProperties_2023-11-14-22-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Fix an issue where PatchPropertiesCorrespondToPutProperties rule checks if the path has both PUT and PATCH calls.",
```

Comment:
```suggestion
      "comment": "Fix an issue where the PatchPropertiesCorrespondToPutProperties rule would fail if either PATCH or PUT operations were undefined.",
```